go.mod

```diff
@@ -5,10 +5,10 @@ go 1.22.0
 toolchain go1.22.5
 
 require (
-	github.com/DataWorkflowServices/dws v0.0.1-0.20250312162549-a682dcdf49b1
+	github.com/DataWorkflowServices/dws v0.0.1-0.20250410182135-484719f8c390
 	github.com/NearNodeFlash/lustre-fs-operator v0.0.1-0.20250219194350-45aad5cbbe17
 	github.com/NearNodeFlash/nnf-ec v0.0.1-0.20250312170301-6f4fd32b0173 // indirect
-	github.com/NearNodeFlash/nnf-sos v0.0.1-0.20250408183859-f6821f7a9ff4
+	github.com/NearNodeFlash/nnf-sos v0.0.1-0.20250411181259-277943ae851b
```

Comment:
Re-vendor after [sos goes in.](https://github.com/NearNodeFlash/nnf-sos/pull/479) goes in.